.github/workflows/ai-review-mcp.yml

```
@@ -0,0 +1,55 @@
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  ai-review:
+    runs-on: ubuntu-latest
+    # APIキーが未設定の場合でも続行
+    continue-on-error: true
+    if: ${{ secrets.ANTHROPIC_API_KEY != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)